tests/next_tests/integration_tests/feature_tests/ffront_tests/test_import_from_mod.py

```diff
@@ -0,0 +1,36 @@
+# GT4Py - GridTools Framework
+#
+# Copyright (c) 2014-2024, ETH Zurich
+# All rights reserved.
+#
+# Please, refer to the LICENSE file in the root directory.
+# SPDX-License-Identifier: BSD-3-Clause
+
+from tests.next_tests.integration_tests.feature_tests.ffront_tests import dimensions_mod as dims
+from tests.next_tests.integration_tests.cases import IDim, KDim
+import gt4py.next as gtx
+import numpy as np
+from tests.next_tests.integration_tests import cases
+from tests.next_tests.integration_tests.cases import cartesian_case
+
+from tests.next_tests.integration_tests.feature_tests.ffront_tests.ffront_test_utils import (
+    exec_alloc_descriptor,
+    mesh_descriptor,
+)
```

Comment:
```suggestion
from next_tests.integration_tests import cases
from next_tests.integration_tests.cases import cartesian_case

from next_tests.integration_tests.feature_tests.ffront_tests.ffront_test_utils import (
    exec_alloc_descriptor,
    mesh_descriptor,
)
```
the top-level tests doesn't have an `__init__.py` (not sure why, but maybe there was a reason)